packages/common/src/initiative-templates/fetch.ts

```diff
@@ -82,6 +85,7 @@ export async function enrichInitiativeTemplateSearchResult(
       thumbnail: "not-implemented",
       workspaceRelative: "not-implemented",
     },
+    location: deriveLocationFromItem(item),
```

Comment:
Thank you for adding this, heads up that `enrichInitiativeTemplateSearchResult` is _never called_. We should let collaboration know so they can add a switch statement to `itemToSearchResult()`